### YamlMime:ManagedReference
items:
- uid: System.Transactions.TransactionInDoubtException
  id: TransactionInDoubtException
  children:
  - System.Transactions.TransactionInDoubtException.#ctor
  - System.Transactions.TransactionInDoubtException.#ctor(System.String)
  - System.Transactions.TransactionInDoubtException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Transactions.TransactionInDoubtException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: TransactionInDoubtException
  nameWithType: TransactionInDoubtException
  fullName: System.Transactions.TransactionInDoubtException
  type: Class
  summary: "A kétséges tranzakció kísérletet egy műveletet, vagy a tranzakció és a tranzakció véglegesítése tett kísérlet történt kivétel InDoubt válik."
  remarks: "A kivétel történt, amikor művelet kísérletet, amely a kétséges tranzakció. A tranzakció kimenetele kétséges, ha a tranzakció állapota nem határozható meg. Pontosabban a tranzakció végső eredményét megmutatják a véglegesítések számát, vagy megszakítja, hogy soha nem ismert ebben a tranzakcióban.       A kivétel is történt kísérlet a tranzakció véglegesítése és a tranzakció InDoubt válik.       Ez az egy helyreállíthatatlan hiba."
  syntax:
    content: 'public class TransactionInDoubtException : System.Transactions.TransactionException'
  inheritance:
  - System.Exception
  - System.SystemException
  - System.Transactions.TransactionException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Transactions.TransactionInDoubtException.#ctor
  id: '#ctor'
  parent: System.Transactions.TransactionInDoubtException
  langs:
  - csharp
  name: TransactionInDoubtException()
  nameWithType: TransactionInDoubtException.TransactionInDoubtException()
  fullName: System.Transactions.TransactionInDoubtException.TransactionInDoubtException()
  type: Constructor
  assemblies:
  - System.Transactions
  namespace: System.Transactions
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Transactions.TransactionInDoubtException&quot;> </xref> osztály."
  syntax:
    content: public TransactionInDoubtException ();
    parameters: []
  overload: System.Transactions.TransactionInDoubtException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Transactions.TransactionInDoubtException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Transactions.TransactionInDoubtException
  langs:
  - csharp
  name: TransactionInDoubtException(String)
  nameWithType: TransactionInDoubtException.TransactionInDoubtException(String)
  fullName: System.Transactions.TransactionInDoubtException.TransactionInDoubtException(String)
  type: Constructor
  assemblies:
  - System.Transactions
  namespace: System.Transactions
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Transactions.TransactionInDoubtException&quot;> </xref> a megadott üzenet osztályra."
  syntax:
    content: public TransactionInDoubtException (string message);
    parameters:
    - id: message
      type: System.String
      description: "A <xref:System.String>, amely tartalmaz egy üzenetet, amely azt ismerteti, miért a kivétel történt.</xref:System.String>"
  overload: System.Transactions.TransactionInDoubtException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Transactions.TransactionInDoubtException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.Transactions.TransactionInDoubtException
  langs:
  - csharp
  name: TransactionInDoubtException(SerializationInfo,StreamingContext)
  nameWithType: TransactionInDoubtException.TransactionInDoubtException(SerializationInfo,StreamingContext)
  fullName: System.Transactions.TransactionInDoubtException.TransactionInDoubtException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System.Transactions
  namespace: System.Transactions
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Transactions.TransactionInDoubtException&quot;> </xref> a megadott szerializálási az osztály és adatfolyam-környezet adatainak."
  syntax:
    content: protected TransactionInDoubtException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "A <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref> objektum, amely egy sikertelen szerializálási ismerteti."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "A <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref> objektum, amely leírja egy sikertelen szerializálási környezetben."
  overload: System.Transactions.TransactionInDoubtException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Transactions.TransactionInDoubtException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Transactions.TransactionInDoubtException
  langs:
  - csharp
  name: TransactionInDoubtException(String,Exception)
  nameWithType: TransactionInDoubtException.TransactionInDoubtException(String,Exception)
  fullName: System.Transactions.TransactionInDoubtException.TransactionInDoubtException(String,Exception)
  type: Constructor
  assemblies:
  - System.Transactions
  namespace: System.Transactions
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Transactions.TransactionInDoubtException&quot;> </xref> osztály a megadott üzenet és a belső kivételben találhatók."
  syntax:
    content: public TransactionInDoubtException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "A <xref:System.String>, amely tartalmaz egy üzenetet, amely azt ismerteti, miért a kivétel történt.</xref:System.String>"
    - id: innerException
      type: System.Exception
      description: "Lekérdezi a kivétel példányon, amely az aktuális kivételt okoz. További információkért lásd: a <xref:System.Exception.InnerException*>tulajdonság.</xref:System.Exception.InnerException*>"
  overload: System.Transactions.TransactionInDoubtException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Transactions.TransactionException
  isExternal: false
  name: System.Transactions.TransactionException
- uid: System.Transactions.TransactionInDoubtException.#ctor
  parent: System.Transactions.TransactionInDoubtException
  isExternal: false
  name: TransactionInDoubtException()
  nameWithType: TransactionInDoubtException.TransactionInDoubtException()
  fullName: System.Transactions.TransactionInDoubtException.TransactionInDoubtException()
- uid: System.Transactions.TransactionInDoubtException.#ctor(System.String)
  parent: System.Transactions.TransactionInDoubtException
  isExternal: false
  name: TransactionInDoubtException(String)
  nameWithType: TransactionInDoubtException.TransactionInDoubtException(String)
  fullName: System.Transactions.TransactionInDoubtException.TransactionInDoubtException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Transactions.TransactionInDoubtException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Transactions.TransactionInDoubtException
  isExternal: false
  name: TransactionInDoubtException(SerializationInfo,StreamingContext)
  nameWithType: TransactionInDoubtException.TransactionInDoubtException(SerializationInfo,StreamingContext)
  fullName: System.Transactions.TransactionInDoubtException.TransactionInDoubtException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.Transactions.TransactionInDoubtException.#ctor(System.String,System.Exception)
  parent: System.Transactions.TransactionInDoubtException
  isExternal: false
  name: TransactionInDoubtException(String,Exception)
  nameWithType: TransactionInDoubtException.TransactionInDoubtException(String,Exception)
  fullName: System.Transactions.TransactionInDoubtException.TransactionInDoubtException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Transactions.TransactionInDoubtException.#ctor*
  parent: System.Transactions.TransactionInDoubtException
  isExternal: false
  name: TransactionInDoubtException
  nameWithType: TransactionInDoubtException.TransactionInDoubtException
